week3/homework/src/actions.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+const fs = require('fs');
+const uuidv4 = require('uuid/v4');
+
+const HELP_FILE_PATH = __dirname + '/help.json';
+const LIST_FILE_PATH = __dirname + '/list.json';
+const DEFAULT_ENCODING = 'utf8';
+
+class ErrorWithStatus extends Error {
+  constructor(errorMessage, statusCode) {
+    super(errorMessage);
+    this.name = 'ErrorWithStatus';
+    this.statusCode = statusCode;
+  }
+}
+
+function findToDoWithId(id) {
```

Comment:
some of these function do not seem to be actions. Perhaps they deserve to be in their own file?
This file is already 150 lines long :(